vm/vmcore/guestmem/src/lib.rs

```diff
@@ -1895,6 +1901,7 @@ pub trait MemoryRead {
 pub trait MemoryWrite {
     fn write(&mut self, data: &[u8]) -> Result<(), AccessError>;
     fn zero(&mut self, len: usize) -> Result<(), AccessError>;
+    fn fill(&mut self, val: u8, len: usize) -> Result<(), AccessError>;
```

Comment:
If you're gonna add this, then remove `zero`, or at least give it a default implementation.